Sample/ViewController.swift

```diff
@@ -1,8 +1,13 @@
 import UIKit
-import RInAppMessaging
+@testable import RInAppMessaging
```

Comment:
@imaginaris `@testable` is needed in order to use an internal API of `RInAppMessaging`?

Reply:
Yes. It's not a very clean solution but I wanted to provide a proper UX for testers. There's no other way to get information about initialization status of the SDK without modifying public API. But still I have mixed feelings